### DISCUSSION TITLE
请问章节标题怎样加粗啊？

### DISCUSSION THREAD

**Discussion by jpDeng111** [sjtug/SJTUThesis#664] [提问] [answered]
操作系统是Win10
使用的是Latex.sjtu.edu.cn
使用的模板是develop分支
克服了名字生僻字的问题之后，和从前的SJTUThesis1.0.0相比章节标题似乎没有加粗，应该怎么办呢？

**Comment by AlexaraWu** [comment, marked as answer]
[上海交通大学关于本科生毕业设计（论文）工作的指导意见.pdf](https://github.com/sjtug/SJTUThesis/files/6505296/default.pdf)

教务处给的撰写规范没有要求章节标题加粗，所以 `develop` 分支把加粗去掉了。
这里不用参考 Word 模板，Word 模板并不符合规范。

**Reply by jpDeng111** [reply]
嗯好的，谢谢Alexara！如今毕业设计走到了尾声，感谢你们一直以来默默地付出！
